src/components/Firestore/api.tsx

```diff
@@ -36,16 +37,22 @@ export default class DatabaseApi implements FirestoreApi {
     this.cleanup = cleanup;
 
     this.baseUrl = `http://${config.hostAndPort}/v1/projects/${projectId}/databases/${databaseId}/`;
+    this.baseEmulatorUrl = `http://${config.hostAndPort}/emulator/v1/projects/${projectId}/databases/${databaseId}/`;
   }
 
   delete(): Promise<void> {
     return this.cleanup();
   }
 
-  private async restPost(path: string, jsonBody: {}) {
+  private async restPost(
```

Comment:
nit: Rename to `restRequest` since it's not just post and also make `method` required to be explicit.

Reply:
Done